Assets/Stronghold_State.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
```

Comment:
Nitpick: It's not important, but if we're not using these, I think we can remove them.